src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -177,6 +180,23 @@ public override void Trace(TraceEvent traceEvent)
             }
         }
 
+        internal static string GetTracePrefix(TraceEvent traceEvent, LogType logType)
+        {
+            // format the trace line metadata prefix
+            var traceProperties = new List<string>()
+            {
+                traceEvent.Level.ToString()
+            };
+            object value = null;
+            if (logType == LogType.Host && traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyFunctionNameKey, out value))
```

Comment:
Including the function name in each log line of a function specific log file is unnecessarily redundant, so I introduce LogType to allow us to tailor log files as needed.

Reply:
This feels a little expensive for something that will be running so frequently

Reply:
Now using a string builder